lib/qa/authorities/linked_data/config/term_config.rb

```diff
@@ -70,13 +70,31 @@ def term_results_id_ldpath
         Config.config_value(term_results, :id_ldpath)
       end
 
+      # Return results id_predicates
+      # @return [Array<String>] the configured predicate to use to extract the id from the results
+      def term_results_id_predicates
```

Comment:
Since the id can come from multiple paths, this method needs to return an array of predicates.

Reply:
Can we replace with this?
```ruby
@pred_ids ||=
  begin
    pred = Config.predicate_uri(term_results, :id_predicate)
    pred ? [pred] : id_predicates_from_ldpath
  end
```